docs/tempo/website/_index.md

```diff
@@ -8,6 +8,19 @@ aliases:
 
 <p align="center"><img src="logo_and_name.png" alt="Tempo Logo"></p>
 
-Grafana Tempo is an open source, easy-to-use and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki.
+Grafana Tempo is an open source, easy-to-use, and high-volume distributed tracing backend. Tempo is cost-efficient, requiring only object storage to operate, and is deeply integrated with Grafana, Mimir, Prometheus, and Loki. You can use Tempo with any of the open-source tracing protocols, including Jaeger, Zipkin, and OpenTelemetry.
```

Comment:
```suggestion
Grafana Tempo is an open source, easy-to-use, and high-volume distributed tracing backend. Tempo is cost-efficient, and only requires an object storage to operate. Tempo is deeply integrated with Grafana, Mimir, Prometheus, and Loki. You can use Tempo with any of the open-source tracing protocols, including Jaeger, Zipkin, or OpenTelemetry.
```

Reply:
Does a user use more than one of these protocols at the same time?

Reply:
Alternative to keep the grammar in tact: "You can use Tempo with open-source tracing protocols, including Jaeger, Zipkin, and OpenTelemetry."

Reply:
Thank you. I've updated the text per your suggestions.